src/config/mekanism.cfg

```diff
@@ -1,244 +1,699 @@
+
+    # The number of items a Ultimate bin can store.
     I:UltimateBinStorage=262144
+
+    # Internal buffer in Joules of each Ultimate universal cable.
     I:UltimateCableCapacity=320000
+
+    # Insulation value of Ultimate thermodynamic conductor.
     D:UltimateConductorConductionInsulation=100000.0
+
+    # Heat capacity of Ultimate thermodynamic conductor.
     D:UltimateConductorHeatCapacity=1.0
+
+    # Conduction value of Ultimate thermodynamic conductor.
     D:UltimateConductorInverseConduction=5.0
+
+    # Maximum number of Joules a Ultimate energy cube can store.
     D:UltimateEnergyCubeMaxEnergy=1.28E8
+
+    # Output rate in Joules of a Ultimate energy cube.
     D:UltimateEnergyCubeOutput=51200.0
+
+    # Output rate of Ultimate gas tank in mB.
     I:UltimateFluidTankOutput=3200
+
+    # Storage size of Ultimate gas tank in mB.
     I:UltimateFluidTankStorage=112000
+
+    # Output rate of Ultimate gas tank in mB.
     I:UltimateGasTankOutput=2056
+
+    # Storage size of Ultimate gas tank in mB.
     I:UltimateGasTankStorage=512000
+
+    # Maximum number of Joules a Ultimate induction cell can store.
     D:UltimateInductionCellMaxEnergy=5.12E11
+
+    # Maximum number of Joules a Ultimate induction provider can output or accept.
     D:UltimateInductionProviderOutput=3.2768E7
+
+    # Capacity of Ultimate mechanical pipe in mB.
     I:UltimatePipeCapacity=64000
+
+    # Pump rate of Ultimate mechanical pipe in mB/t.
     I:UltimatePipePullAmount=6400
+
+    # Item throughput rate of Ultimate logistical transporter in items/s.
     I:UltimateTransporterPullAmount=64
+
+    # Five times travel speed of Ultimate logistical transporter.
     I:UltimateTransporterSpeed=50
+
+    # Capacity of Ultimate pressurized tube in mB.
     I:UltimateTubeCapacity=16384
+
+    # Pump rate of Ultimate pressurized tube in mB/t.
     I:UltimateTubePullAmount=4096
 }
 
 
 usage {
+    # Energy per operation tick (Joules).
     D:ChemicalCrystallizerUsage=400.0
+
+    # Energy per operation tick (Joules).
     D:ChemicalDissolutionChamberUsage=400.0
+
+    # Energy per operation tick (Joules).
     D:ChemicalInfuserUsage=200.0
+
+    # Energy per operation tick (Joules).
     D:ChemicalInjectionChamberUsage=400.0
+
+    # Energy per operation tick (Joules).
     D:ChemicalWasherUsage=200.0
+
+    # Energy per operation tick (Joules).
     D:CombinerUsage=50.0
+
+    # Energy per operation tick (Joules).
     D:CrusherUsage=50.0
+
+    # Energy per operation tick (Joules).
     D:DigitalMinerUsage=100.0
+
+    # Energy per operation tick (Joules).
     D:ElectricPumpUsage=100.0
+
+    # Energy per operation tick (Joules).
     D:EnergizedSmelterUsage=50.0
+
+    # Energy per operation tick (Joules).
     D:EnrichmentChamberUsage=50.0
+
+    # Energy per operation tick (Joules).
     D:FluidicPlenisherUsage=100.0
+
+    # Energy per operation tick (Joules).
     D:FormulaicAssemblicatorUsage=100.0
     D:GasCentrifugeUsage=100.0
```

Comment:
Has no use/no longer used.

Reply:
Thanks for highlighting unused values, for neatness sake I've removed them. The renaming of `VoiceServer` to `WalkieTalkieServer` tripped me up so thanks especially for highlighting that.